ClientUpdater/Updater.cs

```diff
@@ -215,11 +216,9 @@ public static void CheckForUpdates()
     {
         Logger.Log("Updater: Checking for updates.");
         if (VersionState is not VersionState.UPDATECHECKINPROGRESS and not VersionState.UPDATEINPROGRESS)
-        {
 #pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
             DoVersionCheckAsync();
 #pragma warning restore CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

Comment:
```suggestion
            DoVersionCheckAsync();
```

Reply:
CS4014 is still set to `warning`. This is a very nasty one if you accidently miss it. Not awaiting something that should be awaited can have serious consequences and can cause headaches when trying to track down a threading issue. I would caution about disabling this warning and advise against it.
In the given case it is explicitly meant not to be awaited and to be a fire and forget action.
Although `pragma warning` statements are not pretty to look at, this is one of the cases I feel they deserve to have their place.
But if you insist I will set the rule to `suggestion` and remove the `pragma warning`.

Reply:
Right, fair enough, I missed that it wasn't set to `suggestion`.